src/github/client/installation-id.ts

```diff
@@ -0,0 +1,45 @@
+import envVars from "../../config/env";
+
+/**
+ * An installation ID uniquely identifies an installation of a GitHub app across the (single) cloud instance
+ * and (potentially many) GHE instances.
+ */
+export class InstallationId {
```

Comment:
TBH I find it pretty weird to have a class called "Id" which contains some URL... not saying it collides with the current installationIds we have around. I was sure that this was some wrapper thin type/alias before actually reading this code....

Reply:
Yeah, I struggled with the name a bit, too. 

We'll need a new domain object `GitHubApp` when introducing GHE, which will contain the URL and app ID. Once we have that, we should probably split this up into two objects: a (numeric) `installationId` and a `GitHubApp` that contains the `url` and the `appId`

from https://hello.atlassian.net/wiki/spaces/PF/pages/1446606916/GHE+Task+Breakdown: 

![image](https://user-images.githubusercontent.com/1363578/148164612-a49369da-ab18-4772-a40d-b70f8fac0de7.png)